### ISSUE TITLE
Some Simple UI Improvements

### ISSUE THREAD

**Issue by OnyxJet** [Malopieds/InnerTune#359]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

These changes are intended to make the music player screen a bit cleaner looking.

Change contains:
- Moved Icons
- Removal of the oddly placed icons above the progress bar.
- Sleep timer relocation
- Share button relocation

### Why do you want this feature?

![IMG_20240924_100745](https://github.com/user-attachments/assets/155a9bc6-af86-4655-ac04-da6c55956584)

Here you can see the buttons above the progress bar were removed. This is because having them here and within the triple-dot menu is a bit redundant and unnecessary. Also the triple-dot menu was moved to the bottom of the player screen.

The change here is that those buttons have been removed.

Also the share button is now placed next to the queue arrow and the relocated three-dot menu.

But what about the sleep timer button? It never had a place within the triple-dot menu, so now it's gone?

![IMG_20240918_000012](https://github.com/user-attachments/assets/7c200659-d7a6-4dd0-8173-046a720a08c2)

Now the sleep timer button has a place within the triple-dot menu. Infact it takes the place of where the share button used to be.

### Additional information

The point of this all:
The reason you can remove the buttons above the progress bar in the first place is because we already have them in the triple-dot menu (minus the sleep timer, but I moved that to the menu as seen above so it's all good) which is rather redundant. 

**These changes are meant to make the player screen look cleaner and more balanced.**

Notes:
_I used a photo editor app to create this example._

_The sleep timer icon looks **close** but not exactly like your original. That's not me suggesting you should change it to look like mine, I just wasn't able to 100% replicate your icon from scratch. It's all about the location change, not the shape. This also applies to the bottom buttons below the pause/play button._

_The song title and artist/author text was increased to better accommodate the more space. Do note the album cover and progress bar can be adjusted if necessary. I tried to use a font for the text as similar as possible to the original._

The current look of innertunes player for reference. (Though you know this of course :) )

![Screenshot_2024-09-24-10-23-52-00_56635af9b6ad4512d4187a5a49185a25](https://github.com/user-attachments/assets/367b5f24-b328-423d-980b-674dbc10b13b)

**Comment by itsmeteddyhere** [comment]
I think the app looks great as is, and I feel the changes you made would leave a lot of empty space and then if you tried to fill it by making elements bigger it would just look bad.

**Comment by OnyxJet** [comment]
I dont think the extra space is bad at all. Plus the icons can just be increased by maybe 10% to help balance that. The main reason I have suggested all of this is to remove redundancies and remove those awkward buttons above the progress bar. They dont quite fit.

**Comment by Rafee-M** [comment]
> I think the app looks great as is, and I feel the changes you made would leave a lot of empty space and then if you tried to fill it by making elements bigger it would just look bad.

I agree with you on this. I already feel like the seekbar is a bit big lol
